maat-court-data-api/src/main/java/gov/uk/courtdata/assessment/controller/FinancialAssessmentController.java

```diff
@@ -33,9 +42,8 @@ public class FinancialAssessmentController {
     @GetMapping(value = "/{financialAssessmentId}", produces = MediaType.APPLICATION_JSON_VALUE)
     @Operation(description = "Retrieve a financial assessment record")
     @ApiResponse(responseCode = "200", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE))
-    @ApiResponse(responseCode = "400", description = "Bad Request.", content = @Content(mediaType = "application/json", schema = @Schema(implementation = ErrorDTO.class)))
-    @ApiResponse(responseCode = "500", description = "Server Error.", content = @Content(mediaType = "application/json", schema = @Schema(implementation = ErrorDTO.class)))
-    public ResponseEntity<Object> getAssessment(@PathVariable int financialAssessmentId) {
+    @StandardApiResponse
```

Comment:
I believe the StandardApiResponse annotation doesn't contain a 404 - might be worth adding here?

Reply:
I see where you are coming from, I think the reason why a 404 is not included is because the annotation is also used by various create/post endpoints, for which I don't think a 404 would be appropriate.
I'm confident there is more we can do to reduce duplication of boiler-plate annotations and am open to suggestions :)

Reply:
Ah sorry I meant whether it would be worth adding a line in this controller specifically for the 404 (not inside the StandardApiResponse annotation). I did it [here](https://github.com/ministryofjustice/laa-maat-court-data-api/blob/e0326a83e57317d61a55e1bd90e0d7124b3f0345/maat-court-data-api/src/main/java/gov/uk/courtdata/reporder/controller/RepOrderEquityController.java#L29) the other day for something similar. 

There is a bit of duplication doing it this way, but just helps a bit. Whether this is the best implementation I'm not sure :)

Reply:
I have introduced a new annotation class `NotFoundApiResponse.java` and replaced all existing 404 `@ApiResponse` annotations with the new annotation (via search replace and manual review)